lib.ml

```diff
@@ -1214,6 +1270,12 @@ let env : (string * (located_primitive * Types.datatype * pure)) list = [
     datatype ("(String) ~> Int "),
     PURE));
 
+  ("strcontains",
```

Comment:
"convention" is disputable (see e.g. `fold_left`) but since the convention for JS is camelCase, that would be a good way to go